src/scripting/lua_game.cc

```diff
@@ -1499,18 +1502,24 @@ const Widelands::Message& LuaInboxMessage::get(lua_State* L, Widelands::Game& ga
 /* RST
 .. function:: report_result(plr, result[, info = ""])
 
-   Reports the game ending to the metaserver if this is an Internet
-   network game. Otherwise, does nothing.
+   Shows the window indicating the end of the game. The window contains a
```

Comment:
I'm not sure this is correct. I believe it notifies the game of the result for 1 player, and the window is shown when a result has been reported for every player.

Reply:
Chnaged this so it should be more clear